var/spack/repos/builtin/packages/petsc/package.py

```diff
@@ -66,11 +65,13 @@ class Petsc(Package):
     variant('boost',   default=True,  description='Activates support for Boost')
     variant('hypre',   default=True,
             description='Activates support for Hypre (only parallel)')
-    variant('mumps',   default=True,
```

Comment:
could you please add a comment above this line why it's disabled by default so that others won't try to enable it when modifying `petsc` package next time.